records/2021-09-20.scalibility.notify-short-code.md

```diff
@@ -87,22 +87,37 @@ This option could be considered in the future for provincial or territorial gove
 1. The issue around the STOP keyword (and a much needed unsubscribe option) is one that we already have with our current setup and hence, short code will not make it worst (nor improve it).
 
-## Update
+## Update 2021
 
 After discussions with AWS, we considered the following limitations to the short code program in Canada:
 
-- One short code for closely related use cases:
-It is highly unlikely that telco carriers will approve one single short code  for all federal programs. Ideally, uses cases sharing  same short code are logically related, connected and target same subscriber-base. For example, ArriveCAN and Travel Advisory messages for travelers. It is recommended  to highlight commonality of use cases sharing same short code in the application
+- **One short code for closely related use cases**: It is highly unlikely that telco carriers will approve one single short code  for all federal programs. Ideally, user cases sharing same short code are logically related, connected and target same subscriber-base. For example, ArriveCAN and Travel Advisory messages for travelers. It is recommended to highlight commonality of use cases sharing same short code in the application
+
+- **It is recommended to include the following details in the application document**: Detailed description of opt-in process for each use case: screenshots, approval process, enforcement process SMS message templates: constant part  and variable parts of messages who does control and  fill-in the template for each use case (GCNotify team or GoC departments). This information is required by carriers to understand who is a contact person/team if quick updates is needed.
+
+Given that there is a strong preference for one short code only for closely related use cases, and given that it is recommended to include detailed documentation for each use case, the financial and administrative costs at this time are too high for us to proceed with our application.
+
+We will reconsider opening our file for a short code, or multiple shortcode if the need arises and the use case is a good fit for the program and parameters given by AWS, CWTA and the carriers.
+
+## Update 2024
+
+We revived the effort in 2023 due to our scaling need for SMS, coming as a requirement for getting provincial beta onboarding program within GCNotify.
+
+A policy research was performed to dig into CASL legislation and see if that should apply to governmental bodies. The findings were that it should not. Considering this and after talks with AWS (who talked to aggregators), it was deemeed that it would be a safe strategy to go forward with a specialized use case for a new short code request. This would get us a foot in the door, while looking for options to upgrade our existing short code to a generic usage once we got it by trying to get in touch with the CWTA organization.
```

Comment:
```suggestion
A policy research was performed to dig into CASL legislation and see if that should apply to governmental bodies. The findings were that it should not. Considering this and after talks with AWS (who talked to aggregators), it was deemeed that it would be a safe strategy to go forward with a specialized use case for a new short code request. This would get us a foot in the door, while looking for options to upgrade our existing short code to a generic usage once we got it by trying to get in touch with the CTA organization.
```